frontend/src/LandingPage.js

```diff
@@ -1,50 +1,269 @@
+                if (user.loggedIn) {
+                    loginSection.style.display = 'none';
+                    logoutMessage.href = user.url;
+                    logoutSection.style.display = 'block';
+                    
+                } else {
+                    logoutSection.style.display = 'none';
+                    loginMessage.href = user.url;
+                    loginSection.style.display = 'block';
+                    
+                }
+                currentUserId = user.id;
+                localStorage.setItem("id", currentUserId);
+            });
+    }
+
+    function calls() {
+        handleScriptLoad1(setQuery1);
+        handleScriptLoad2(setQuery2);
+    }
+
+    const loadScript = (url) => {
+        let script = document.createElement("script");
+        script.type = "text/javascript";
+
+        if (script.readyState) {
+            script.onreadystatechange = function() {
+                if (script.readyState === "loaded" || script.readyState === "complete") {
+                    script.onreadystatechange = null;
+                    calls();
+                }
+            };
+        } else {
+            script.onload = () => calls();
+        }
+
+        script.src = url;
+        document.getElementsByTagName("head")[0].appendChild(script);
+    };
+    function handleScriptLoad1(updateQuery) {
+        autoComplete1 = new window.google.maps.places.Autocomplete(
+            document.getElementById("startplace"), { "types": ["geocode"] }
+        );
+        autoComplete1.setFields(["address_components", "formatted_address"]);
+        autoComplete1.addListener("place_changed", () =>
+            handlePlaceSelect1(updateQuery)
+        );
+    }
+    function handleScriptLoad2(updateQuery) {
+        autoComplete2 = new window.google.maps.places.Autocomplete(
+            document.getElementById("endplace"), { "types": ["geocode"] }
+        );
+        autoComplete2.setFields(["address_components", "formatted_address"]);
+        autoComplete2.addListener("place_changed", () =>
+            handlePlaceSelect2(updateQuery)
+        );
+    }
+
+    async function handlePlaceSelect1(updateQuery) {
+        let addressObject1 = autoComplete1.getPlace();
+        console.log(addressObject1);
+        //lat1 = addressObject1.geometry.location.lat();
+        //long1 = addressObject1.geometry.location.lng();
+        let query1 = addressObject1.formatted_address;
+        updateQuery(query1);
+        
+
+    }
+
+    async function handlePlaceSelect2(updateQuery) {
+        const addressObject2 = autoComplete2.getPlace();
+        console.log(addressObject2);
+        //lat2 = addressObject2.geometry.location.lat();
+        //long2 = addressObject2.geometry.location.lng();
+        const query2 = addressObject2.formatted_address;
+        updateQuery(query2);
+    }
+
+    const [query1, setQuery1] = useState("");
+    const [query2, setQuery2] = useState("");
+
+    // useEffect(() => {
```

Comment:
Please remove commented out lines of code here and above

Reply:
Fixed, thanks.